includes/libraries/omise-plugin/helpers/charge.php

```diff
@@ -35,9 +35,7 @@ public static function isAuthorized($charge)
         public static function isPaid($charge)
         {
             if (self::isChargeObject($charge)) {
-                // support Omise API version '2014-07-27' by checking if 'captured' exist.
-                $paid = isset($charge['captured']) ? $charge['captured'] : $charge['paid'];
-                if ($paid === true)
+                if ($charge['paid'] === true)
```

Comment:
Code feels awfully verbose/inefficient. Why not just a one liner?

`return self::isChargeObject($charge) && $charge['paid'];`

Reply:
👍